audit/projects/k8s-infra-prow-build-trusted/services/container/clusters/prow-build-trusted.json

```diff
@@ -20,8 +20,8 @@
   "binaryAuthorization": {},
   "clusterIpv4Cidr": "10.4.0.0/14",
   "createTime": "2020-04-30T23:44:46+00:00",
-  "currentMasterVersion": "1.21.6-gke.1503",
-  "currentNodeVersion": "1.20.10-gke.1600 *",
+  "currentMasterVersion": "1.21.9-gke.1002",
```

Comment:
manually triggered to catch a fix for GKE. https://github.com/kubernetes/k8s.io/issues/3459#issuecomment-1063754253